.travis.yml

```diff
@@ -0,0 +1,244 @@
+  - os: linux
+    compiler: gcc
+    addons: &9
+      apt:
+        sources:
+        - ubuntu-toolchain-r-test
+        - george-edison55-precise-backports
+        packages:
+        - cmake
+        - cmake-data
+        - liblapack-dev
+        - g++-5
+        - gcc-5
+        - gfortran-5
+    env: CXX_COMPILER='g++-5' C_COMPILER='gcc-5' Fortran_COMPILER='gfortran-5' BUILD_TYPE='release' BOOSTSTR="--boost-incdir=${TRAVIS_BUILD_DIR}/boost_build/boost_1_57_0 --boost-libdir=${TRAVIS_BUILD_DIR}/boost_build/boost_1_57_0/stage/lib" BOOSTBUILD=".scripts/build_boost.sh 5" OMP_NUM_THREADS=2
+  - os: linux
+    compiler: gcc
+    addons: *9
+    env: CXX_COMPILER='g++-5' C_COMPILER='gcc-5' Fortran_COMPILER='gfortran-5' BUILD_TYPE='debug' BOOSTSTR="--boost-incdir=${TRAVIS_BUILD_DIR}/boost_build/boost_1_57_0 --boost-libdir=${TRAVIS_BUILD_DIR}/boost_build/boost_1_57_0/stage/lib" BOOSTBUILD=".scripts/build_boost.sh 5" OMP_NUM_THREADS=2
+
+#  - os: osx
+#    osx_image: xcode6.4
+#    compiler: clang
+#    env: CXX_COMPILER='clang++' C_COMPILER='clang' Fortran_COMPILER='gfortran' BUILD_TYPE='debug'
+#  - os: osx
+#    osx_image: xcode6.4
+#    compiler: clang
+#    env: CXX_COMPILER='clang++' C_COMPILER='clang' Fortran_COMPILER='gfortran' BUILD_TYPE='release'
+#  - os: osx
+#    osx_image: xcode7
+#    compiler: clang
+#    env: CXX_COMPILER='clang++' C_COMPILER='clang' Fortran_COMPILER='gfortran' BUILD_TYPE='debug'
+#  - os: osx
+#    osx_image: xcode7
+#    compiler: clang
+#    env: CXX_COMPILER='clang++' C_COMPILER='clang' Fortran_COMPILER='gfortran' BUILD_TYPE='release'
+#  - os: osx
+#    osx_image: xcode6.4
+#    compiler: gcc
+#    env: CXX_COMPILER='g++-5' C_COMPILER='gcc-5' Fortran_COMPILER='gfortran' BUILD_TYPE='debug'
+#  - os: osx
+#    osx_image: xcode6.4
+#    compiler: gcc
+#    env: CXX_COMPILER='g++-5' C_COMPILER='gcc-5' Fortran_COMPILER='gfortran' BUILD_TYPE='release'
+#  - os: osx
+#    osx_image: xcode7
+#    compiler: gcc
+#    env: CXX_COMPILER='g++-5' C_COMPILER='gcc-5' Fortran_COMPILER='gfortran' BUILD_TYPE='debug'
+#  - os: osx
+#    osx_image: xcode7
+#    compiler: gcc
+#    env: CXX_COMPILER='g++-5' C_COMPILER='gcc-5' Fortran_COMPILER='gfortran' BUILD_TYPE='release'
+install:
+- DEPS_DIR="${TRAVIS_BUILD_DIR}/deps"
+- mkdir -p ${DEPS_DIR} && cd ${DEPS_DIR}
+- |
+  if [[ "${TRAVIS_OS_NAME}" == "osx" ]]; then
+    brew install cmake python
+    brew install boost --with-python
+    brew install boost-python
+  fi
+before_script:
+- cd ${TRAVIS_BUILD_DIR}
+- export CXX=${CXX_COMPILER}
+- export CC=${C_COMPILER}
+- export FC=${Fortran_COMPILER}
+- ${BOOSTBUILD}
+- python setup --cxx=${CXX_COMPILER} --cc=${C_COMPILER} --fc=${Fortran_COMPILER} --type=${BUILD_TYPE} ${BOOSTSTR} --max-am-eri=4 --chemps2=off objdir
+- cd objdir
+script:
+- ../.scripts/travis_build.sh
```

Comment:
Thanks for the suggestion, Roberto.  I'm happy for everything to go in the YAML, and would greatly prefer that.  One of the (many) earlier versions of the script did that, but I found the tests were running before the build.  Maybe I had screwed something else up.  I'll play around some more and see what I can figure out.